### DISCUSSION TITLE
Why does the code stream GOP structure obtained after encoding contain only frame I and frame B？

### DISCUSSION THREAD

**Discussion by hello905** [fraunhoferhhi/vvenc#472] [General]
Like this!
![1732532135638-cxv](https://github.com/user-attachments/assets/c943af1f-78e4-4ef7-863d-13fa92352431)
The command is './vvencapp -s 128x128 --framerate 60 -c yuv420 -ip 10 -i 128_128_8.yuv --hdr hlg_2020 -o 128_128_8.vvc'

**Comment by adamjw24** [comment]
Because P frames are bad and a relict of the 90s. Why not? What would thing is better?
